### ISSUE TITLE
[Tests]: CI tests to check code works properly

### ISSUE THREAD

**Issue by Horizon733** [Horizon733/certi-build#1]
- [ ] tests to verify certificates saved
- [ ] test for extracting names from excel sheet
- [ ] test to vallidate hex color codes

**Comment by AvineshTripathi** [comment]
I was willing to work on this issue can you assign me 
also are you planning to use linter and which tool are you using for CI

**Comment by Horizon733** [comment]
> I was willing to work on this issue can you assign me also are you planning to use linter and which tool are you using for CI

Good to see you want to contribute, so currently this tool repo is new and I will be using CI for workflows. So, you can write tests which work with Circle CI workflow

**Comment by AvineshTripathi** [comment]
I ll do it then

**Comment by AvineshTripathi** [comment]
hey @Horizon733 I m not able to setup the repo in my local can you guide me for the same

**Comment by Horizon733** [comment]
> hey @Horizon733 I m not able to setup the repo in my local can you guide me for the same

Hi, what's the issue you are facing

**Comment by AvineshTripathi** [comment]
M getting 2 error in my builder.py also do we have to make separate png, exel file and ttf file ?

**Comment by Horizon733** [comment]
> M getting 2 error in my builder.py also do we have to make separate png, exel file and ttf file ?

Yes while generating a certificate, you will need png, excel and ttf file. If you have a look at README there I have mentioned this.
Also, please add everything while asking for a solution for error as follows.
OS:
Python version
error stacktrace
screenshots of errors
And other relevant information

**Comment by AvineshTripathi** [comment]
![Screenshot from 2021-10-02 11-57-12](https://user-images.githubusercontent.com/73980067/135706177-27c3db6a-3ad1-4947-962e-116da69aec86.png)

(error stacktrack)

![Screenshot from 2021-10-02 11-59-39](https://user-images.githubusercontent.com/73980067/135706206-c2acb41d-6672-47c6-9aa0-84221f0278e4.png)


(builder.py)
Python 3.8.10(python version )


-> also you have not mentioned the point of downloading dependencies from requirements.txt

**Comment by Horizon733** [comment]
> ![Screenshot from 2021-10-02 11-57-12](https://user-images.githubusercontent.com/73980067/135706177-27c3db6a-3ad1-4947-962e-116da69aec86.png)
> 
> (error stacktrack)
> 
> ![Screenshot from 2021-10-02 11-59-39](https://user-images.githubusercontent.com/73980067/135706206-c2acb41d-6672-47c6-9aa0-84221f0278e4.png)
> 
> (builder.py) Python 3.8.10(python version )
> 
> -> also you have not mentioned the point of downloading dependencies from requirements.txt

Thanks for raising this issue, and I have updated the readme and could please open another issue focusing on the first screenshot. It would be helpful for people who want to contribute and link their pr with that issue. Make sure to add all details for error.

**Comment by AvineshTripathi** [comment]
issue resolved started working on CI test on circleci
